### ISSUE TITLE
Identification of Meetups

### ISSUE THREAD

**Issue by abhansnuk** [wpmarketingteam/Marcomms-Meetups#8]

**Comment by TantienHime** [comment]
Similar to this, @vimes1984 and Cami were discussing having a channel where we could "stream" online meetups going on 24/7. I was supposed to send Cami an email about it, but have yet to do it. We also looked into finding local meetups as part of the Diversity team to identify meetups in our local area on meetup.com. Depending on what you want to accomplish with this, I may be able to help.

**Comment by murillol** [comment]
> Similar to this, @vimes1984 and Cami were discussing having a channel where we could "stream" online meetups going on 24/7. I was supposed to send Cami an email about it, but have yet to do it. We also looked into finding local meetups as part of the Diversity team to identify meetups in our local area on meetup.com. Depending on what you want to accomplish with this, I may be able to help.

The 24/7 channel sounds really cool @TantienHime !! I think this particular issue "identifying meetups" was initially about selecting meetup events that we can promote in our weekly Roundups on social.

**Comment by TantienHime** [comment]
Agreed @murillol. I think the idea of the channel was to populate or broadcast local online meetups. There might be an opportunity to kill two (or even three) birds with one stone. 

This link was provided by Hari Shankar a while back to find the chapter WordPress meetups on meetup.com: [https://www.meetup.com/pro/wordpress](https://www.meetup.com/pro/wordpress)
